cfpb/cfgov-refresh
# Make sub-filterable preview images clickable

Make sub-filterable preview images clickable

## Changes

- Modified `cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html` to pass `should_link_image`.
- Modified `cfgov/v1/atomic_elements/organisms.py` to add the should_link_image boolean.

## Testing

- Visit `http://localhost:8000/consumer-tools/everyone-has-a-story/` in Wagtail.
- Enable the `Should link image` flag and publish. 
- Click the preview image on the sub-filterable landing page and be guided to the appropriate page. 


## Screenshots
<img width="1201" alt="screen shot 2017-02-16 at 3 06 24 pm" src="https://cloud.githubusercontent.com/assets/1696212/23038866/9996de56-f459-11e6-877c-c2b490176eaf.png">


## Notes

- At some point someone will possess the moxie to refactor `cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html`.


## Checklist
